code/datums/mood_events/generic_positive_events.dm

```diff
@@ -107,6 +107,16 @@
 	timeout = 3 SECONDS
 	hidden = TRUE
 
+/datum/mood_event/paranoid_armed
+	description = "I feel more secure with a weapon on me." //Renegades get it when armed
+	mood_change = 4
+	hidden = TRUE
+
+/datum/mood_event/paranoid_armored
+	description = "I feel more secure with armor on me." //Renegades get it when have armor on them
+	mood_change = 2
+	hidden = TRUE
```

Comment:
IMO if these are only used in one place they should just be defined around that place, I don't like these misc files

Reply:
> IMO if these are only used in one place they should just be defined around that place, I don't like these misc files

I can shove it into the Paranoia trauma file